cea/technologies/thermal_network/thermal_network_costs.py

```diff
@@ -0,0 +1,563 @@
+    mdotnMax_kgpers = np.amax(mdotA_kgpers)  # find highest mass flow of all nodes at all timesteps (should be at plant)
+    # read in total pressure loss in kW
+    deltaP_kW = pd.read_csv(network_info.locator.get_ploss('', network_type))
+    deltaP_kW = deltaP_kW['pressure_loss_total_kW'].sum()
+
+    Opex_var = deltaP_kW * 1000 * network_info.prices.ELEC_PRICE
+
+    if network_info.config.thermal_network.network_type == 'DH':
+        deltaPmax = np.max(network_info.network_features.DeltaP_DHN)
+    else:
+        deltaPmax = np.max(network_info.network_features.DeltaP_DCN)
+    Capex_a, Opex_fixed = pumps.calc_Cinv_pump(deltaPmax, mdotnMax_kgpers, PUMP_ETA, network_info.config,
+                                               network_info.locator, 'PU1')  # investment of Machinery
+    Opex_a_tot = Opex_var + Opex_fixed
+
+    return Capex_a, Opex_a_tot
+
+
+def calc_Ctot_cooling_plants(network_info):
+    """
+    Calculates costs of centralized cooling plants (chillers and cooling towers).
+    :param network_info: an object storing information of the current network
+    :return:
+    """
+
+    # read in plant heat requirement
+    plant_heat_hourly_kWh = pd.read_csv(
+        network_info.locator.get_optimization_network_layout_plant_heat_requirement_file(
+            network_info.network_type, network_info.config.thermal_network_optimization.network_name))
+    # read in number of plants
+    number_of_plants = len(plant_heat_hourly_kWh.columns)
+
+    plant_heat_original_kWh = plant_heat_hourly_kWh.copy()
+    plant_heat_peak_kW_list = plant_heat_hourly_kWh.abs().max(axis=0).values  # calculate peak demand
+    plant_heat_sum_kWh_list = plant_heat_hourly_kWh.abs().sum().values  # calculate aggregated demand
+
+    Opex_var_chiller = 0.0
+    Opex_var_CT = 0.0
+    Capex_a_chiller = 0.0
+    Opex_a_chiller = 0.0
+    Capex_a_CT = 0.0
+    Opex_a_CT = 0.0
+    # calculate cost of chiller heat production and chiller capex and opex
+    for plant_number in range(number_of_plants):  # iterate through all plants
+        if number_of_plants > 1:
+            plant_heat_peak_kW = plant_heat_peak_kW_list[plant_number]
+        else:
+            plant_heat_peak_kW = plant_heat_peak_kW_list[0]
+        plant_heat_yearly_kWh = plant_heat_sum_kWh_list[plant_number]
+        print 'Annual plant heat production:', round(plant_heat_yearly_kWh, 0), '[kWh]'
+        Capex_chiller = 0
+        Opex_fixed_chiller = 0
+        Capex_CT = 0
+        Opex_fixed_CT = 0
+        if plant_heat_peak_kW > 0:  # we have non 0 demand
+            peak_demand_W = plant_heat_peak_kW * 1000  # convert to W
+            print 'Calculating cost of heat production at plant number: ', plant_number
+            if network_info.config.thermal_network_optimization.yearly_cost_calculations:
+                # calculates operation costs with yearly approximation
+                COP_plant = VCCModel.calc_VCC_COP(network_info.config,
```

Comment:
After checking the source of the numbers, this COP_plant already include opex_var for chiller, cooling tower, condenser water pumps and chilled water pumps? (please confirm)
Therefore, the equation calculating Opex_var_CT in line 152 is redundant. (sorry, there were probably some misunderstanding when i first requested the change)

If you can confirm that, the output of this function `calc_Ctot_cooling_plants` should be: `Opex_a_cooling_plants`, `Capex_a_CT`, `Capex_a_chiller`.
and `Opex_a_cooling_plants` = `Opex_var_plant` + `Opex_fixed_CT` + `Opex_fixed_chiller`

Reply:
I checked the source and you are right - the calculation is based on values which already contain the CT and chiller opex. I am implementing the changes as you suggested.

Reply:
What about the Opex_var_CT? The way I understand this, it is also included in the Opex_a_cooling_plants. 
Then it would be:
´Opex_a_cooling_plants = Opex_var_plant + Opex_fixed_CT + Opex_fixed_chiller + Opex_var_CT´

Reply:
It depends on what is included in COP_plant
from my understanding, the `Opex_var_plant` calculated from COP_plant includes the operation cost of cooling towers as well, that means we can treat `Opex_var_plant` = `Opex_var_chiller` + `Opex_var_CT`

Reply:
I agree!